Example/Podfile

```diff
@@ -15,7 +15,7 @@ target 'Emission' do
   pod 'Emission', :path => '../'
 
   # As this runs dev, we need the developer web socket
-  pod 'React', :path => react_path, :subspecs => %w(RCTWebSocket)
+  pod 'React', :path => react_path, :subspecs => %w(DevSupport BatchedBridge RCTAnimation)
```

Comment:
I haven't found one. The `Gene` container relies on `react-native-parallax-scroll-view`, which uses `Animated`. However, the `ArtistCard` also uses `Animated` and works fine without `RCTAnimation`, so perhaps there is a way around it? With the `post-install` script and `RCTAnimation`, though, the `Gene` error is solved, so maybe it's a good idea to include it if we can.